src/v2/Apps/Payment/Components/PaymentSection.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Button, Theme } from "@artsy/palette"
+import { SystemContextProps } from "v2/Artsy"
+import { get } from "v2/Utils/get"
+
+import React, { useState } from "react"
+import { RelayProp, createFragmentContainer, graphql } from "react-relay"
+import { SavedCreditCards } from "v2/Components/Payment/SavedCreditCards"
+import { PaymentModal } from "v2/Components/Payment/PaymentModal"
+import { PaymentSection_me } from "v2/__generated__/PaymentSection_me.graphql"
+import { UserSettingsPaymentsCreditCard } from "v2/__generated__/UserSettingsPaymentsCreditCard.graphql"
+import { loadStripe } from "@stripe/stripe-js"
+import { Elements } from "@stripe/react-stripe-js"
+import { data as sd } from "sharify"
+
+interface PaymentSectionProps extends SystemContextProps {
+  relay?: RelayProp
+  me: PaymentSection_me
+}
+
+export const PaymentSection: React.FC<PaymentSectionProps> = props => {
+  const creditCardEdges = get(props, props => props.me.creditCards.edges)
```

Comment:
We don't need the `get` function anymore since we can use JS's existential operator: `foo?.bar?.baz?.()?.[20]?.bam`